alacritty/src/display/window.rs

```diff
@@ -298,7 +299,8 @@ impl Window {
 
     #[cfg(windows)]
     pub fn get_platform_window(_: &Identity, window_config: &WindowConfig) -> WindowAttributes {
-        let icon = winit::window::Icon::from_resource(IDI_ICON, None);
+        let icon: std::result::Result<winit::window::Icon, winit::window::BadIcon> =
```

Comment:
this change is made by vscode automatically...very fast without interruption it might be worth going to not have to deal with it.

Reply:
normal is the default.

tested using local git:
```
cargo run -- -o window.window_level="'AlwaysOnTop'"
```
on top.

I tried
```
cargo run -- -o window.window_level="'AlwaysOnBottom'"
```
and it isn't starting behind the windows.  it did when I was testing via a direct function set_window_level that I removed as deadcode.  I'm not sure what's up there but normal and always on top are working great for me.

Reply:
ubuntu 24.10 doesn't seem to apply the setting...command line parsed into three options, AlwaysOnTop doesn't keep it on top with given code.

Reply:
Are the explicit type annotations even needed? Isn't the return type of `from_resource` enough?

Reply:
that I left in as VS code is doing that automatically.  it can be pulled out; I don't know why vscode feels so strongly about that line.

Reply:
This should be removed.